win32ss/user/rtl/image.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * PROJECT:     ReactOS Win32ss RTL
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
```

Comment:
```suggestion
 * PROJECT:     ReactOS user32.dll
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
```
Based on existing file in this directory: https://github.com/reactos/reactos/blob/master/win32ss/user/rtl/text.c

Reply:
Thanks.